osu.Game.Tournament/Screens/SetupScreen.cs

```diff
@@ -40,6 +42,9 @@ public class SetupScreen : TournamentScreen, IProvideVideo
         [Resolved]
         private RulesetStore rulesets { get; set; }
 
+        [Resolved]
+        private Storage storage { get; set; }
```

Comment:
I'd probably prefer this to be done as such:

```csharp
private TournamentStorage storage;

// in BDL, before reload()
private void load(FrameworkConfigManager frameworkConfig, Storage storage)
{
    // ...
    this.storage = (TournamentStorage)storage;
    reload();
}

// can then remove the access null-checks
```

I think that's much clearer when it comes to the intent (the incoming storage needs to be a tournament storage, and is guaranteed non-null if the hard cast succeeds).